project/iterative_sorting.py

```diff
@@ -33,7 +31,13 @@ def insertion_sort( arr ):
 
 # STRETCH: implement the Bubble Sort function below
 def bubble_sort( arr ):
-
+    is_sorted = False
+    while not is_sorted:
```

Comment:
very nice. one optimization: the way you're running it now, you have to one whole pass through the array before declaring it sorted. can you figure out a way to ensure it's sorted once you've gone through all of it? this can bring down the runtime of your solution by almost half